src/styles/_header.scss

```diff
@@ -0,0 +1,53 @@
+header {
+  .content {
+    min-height: 10em;
+
+    .photo-container,
+    .heading {
+      padding-top: 0;
+    }
+
+    .heading {
+      justify-content: center;
+      margin-left: 1.6em;
+    }
+  }
+}
+
+.photo__img {
+  height: 104px;
+  width: 104px;
+  border-color:$main-colot;
+  border-width: 2px;
+  border-style: solid;
+  border-radius: 50%;
+  background-color: $secondry-color;
+  padding: 10px;
+  box-sizing: border-box;
+  img {
+    border-radius: 50%;
+    height: 80px;
```

Comment:
Try to avoid using px. Please check every where - https://learn.javascript.ru/css-units .